tests/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ add_subdirectory(performance)
 # Microbenchmarks.
 add_subdirectory(ubench)
 
+# regression / delta tests
+# Employing the full simulator. validated using deltas on output data
+add_subdirectory(regression)
```

Comment:
Can you split out the regressions PR contents from this PR? It would make it easier to review…